# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/Document.h
# Commit: 82e03a404c2f
# Full Hash: 82e03a404c2fe76b1240dce4fa3d0b6bde909a87
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2023-12-05 21:29:44
# Regressor Bug: 1589554
# File Overlap Count: 1
# Description:
#   Bug 1589554 - Part 2: Implement Screen Wake Lock API. r=dom-core,edgar
#   
#   Depends on D189508
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D189509
# ==============================================================================

diff -r b55991835aec -r 82e03a404c2f dom/base/Document.h
--- a/dom/base/Document.h	Tue Dec 05 15:37:05 2023 +0000
+++ b/dom/base/Document.h	Tue Dec 05 15:37:05 2023 +0000
@@ -3540,6 +3540,8 @@
 
   nsTHashSet<RefPtr<WakeLockSentinel>>& ActiveWakeLocks(WakeLockType aType);
 
+  void UnlockAllWakeLocks(WakeLockType aType);
+
   // ParentNode
   nsIHTMLCollection* Children();
   uint32_t ChildElementCount();